components/html-block/html-block.js

```diff
@@ -3,6 +3,43 @@ import { css, LitElement } from 'lit-element/lit-element.js';
 import { htmlBlockMathRenderer } from '../../helpers/mathjax.js';
 import { requestInstance } from '../../mixins/provider-mixin.js';
 
+let configuredIfrauClient;
+
+const getIfrauClient = async() => {
+	if (configuredIfrauClient === undefined && window.ifrauclient) {
+		configuredIfrauClient = await window.ifrauclient({
```

Comment:
I think there is a "slim" client that we should be looking into using for our core components that require such a check.  I don't think it's documented though, so I think we might need to look into the code a little to figure out what exactly it does, but I think it is meant to avoid some of the issues we've had with client's clobbering each other.

Reply:
Yeah, that's a good point. If we're only fetching the host we should be able to use that.